### ISSUE TITLE
Error: Returning from a request handler without responding or attaching an abort handler is forbidden!

### ISSUE THREAD

**Issue by alexhultman** [uNetworking/uWebSockets#1180]
This message is the only one we write and the only place where std::terminate is called.

This behavior is a MUST since you otherwise end up in an invalid state where you can end up holding resources for an HTTP socket that has been dead for ages.

However, this is not an elegant interface - the interface should not really allow the user to make this mistake.

Brainstorming for how to solve it more elegantly -

**Comment by alexhultman** [comment]
Hmmm... I think I remember the message was added for the Node.js users. In C++ shooting yourself in the foot should be perfectly allowed, but I think I added this as a quick fix to enforce Node.js users to not end up with native resources leaked or lost.

Maybe a better solution is to come up with some way for the Node.js addon to not end up here - like have the Node.js addon always set onAborted internally even if the script did not do so?

Node.js users don't like std::terminate and std::cerr - they want JS exceptions instead.
